python/GafferArnold/ArnoldTextureBake.py

```diff
@@ -0,0 +1,495 @@
+				parent["renderFileList"] = IECore.StringVectorData( fileList )
+				"""
+			), "python" )
+
+
+
+			self["__parent"] = GafferScene.Parent()
+			self["__parent"]["parent"].setValue( "/" )
+			for c in ['bound', 'transform', 'attributes', 'object', 'childNames', 'setNames', 'set']:
+				self["__parent"]["in"][c].setInput( self["in"][c] )
+
+			self["__outputExpression"] = Gaffer.Expression()
+			self["__outputExpression"].setExpression( inspect.cleandoc(
+				"""
+				import IECoreScene
+				g = parent["in"]["globals"]
+				info = parent["__chunkedBakeInfo"]
+				for cameraName, i in info.items():
+					params = IECore.CompoundData()
+					fileName = i["fileName"].value
+					params["camera"] = IECore.StringData( "/" + parent["cameraGroup"] + "/" + cameraName )
+					for nameAndAov in parent["aovs"].strip( " " ).split( " " ):
+						tokens = nameAndAov.split( ":" )
+						if len( tokens ) != 2:
+							raise RuntimeError( "Invalid bake aov specification: %s It should contain a : between name and data." )
+						( aovName, aov ) = tokens
+						aovFileName = fileName.replace( "<AOV>", aovName )
+						g["output:" + cameraName + "." +  aov] = IECoreScene.Output( aovFileName, "exr", aov + " RGBA", params )
+				parent["__parent"]["in"]["globals"] = g
+				"""
+			), "python" )
+
+			self["__camera"] = GafferScene.Camera()
+			self["__camera"]["projection"].setValue( "orthographic" )
+
+			self["__cameraTweaks"] = GafferScene.CameraTweaks()
+			self["__cameraTweaks"]["in"].setInput( self["__camera"]["out"] )
+			self["__cameraTweaks"]["tweaks"]["projection"] = GafferScene.TweakPlug( "projection", "uv_camera" )
+			self["__cameraTweaks"]["tweaks"]["resolution"] = GafferScene.TweakPlug( "resolution", imath.V2i( 0 ) )
+			self["__cameraTweaks"]["tweaks"]["u_offset"] = GafferScene.TweakPlug( "u_offset", 0.0 )
+			self["__cameraTweaks"]["tweaks"]["v_offset"] = GafferScene.TweakPlug( "v_offset", 0.0 )
+			self["__cameraTweaks"]["tweaks"]["mesh"] = GafferScene.TweakPlug( "mesh", "" )
+			self["__cameraTweaks"]["tweaks"]["uv_set"] = GafferScene.TweakPlug( "uv_set", "" )
+			self["__cameraTweaks"]["tweaks"]["extend_edges"] = GafferScene.TweakPlug( "extend_edges", False )
+
+			self["__cameraTweaksFilter"] = GafferScene.PathFilter()
+			self["__cameraTweaksFilter"]["paths"].setValue( IECore.StringVectorData( [ '/camera' ] ) )
+			self["__cameraTweaks"]["filter"].setInput( self["__cameraTweaksFilter"]["out"] )
+
+
+			self["__collectScenes"] = GafferScene.CollectScenes()
+			self["__collectScenes"]["sourceRoot"].setValue( "/camera" )
+			self["__collectScenes"]["rootNameVariable"].setValue( "collect:cameraName" )
+			self["__collectScenes"]["in"].setInput( self["__cameraTweaks"]["out"] )
+
+			self["__group"] = GafferScene.Group()
+			self["__group"]["in"][0].setInput( self["__collectScenes"]["out"] )
+			self["__group"]["name"].setInput( self["cameraGroup"] )
+
+			self["__parent"]["child"].setInput( self["__group"]["out"] )
```

Comment:
If the input to the baker already has a "bakeCameras" location, our "bakeCameras" will be renamed to "bakeCameras1", and everything will break. Should we use a name that's less likely to clash?